provision/ubuntu/install.sh

```diff
@@ -104,7 +104,7 @@ cd /tmp
 git clone -b master https://github.com/llvm/llvm-project.git llvm
 mkdir -p llvm/llvm/build/install
 cd llvm
-git checkout -b d941df363d1cb621a3836b909c37d79f2a3e27e2 d941df363d1cb621a3836b909c37d79f2a3e27e2
+git checkout -b 176249bd6732a8044d457092ed932768724a6f06 176249bd6732a8044d457092ed932768724a6f06
```

Comment:
Could we use the `llvmorg-11.0.0` tag here to make it more obvious which version we're using?